packages/patternlab/styleguide/source/_patterns/03-organisms/by-author/callout-links-three-up.twig

```diff
@@ -0,0 +1,38 @@
+{% set headingLevel = calloutLinksThreeUp.headingLevel ?: "3" %}
+
+{% set headingText %}
+  {% if calloutLinksThreeUp.headingText is not empty %}
+    {% set heading1 = {"text": calloutLinksThreeUp.headingText } %}
+    {% set heading2 = {"text": calloutLinksThreeUp.headingText } %}
+    {% set heading3 = {"text": calloutLinksThreeUp.headingText } %}
+    {% set heading4 = {"text": calloutLinksThreeUp.headingText } %}
```

Comment:
maybe add heading5 and heading 6 here? 

We should really make the heading data more flexible so it doesn't need this

Reply:
I can create 5 and 6 levels. Good point. Thanks!